src/librustc_data_structures/blake2b.rs

```diff
@@ -113,17 +113,17 @@ fn blake2b_compress(ctx: &mut Blake2bCtx, last: bool) {
     }
 
     {
-        // Re-interpret the input buffer in the state as u64s
+        // Re-interpret the input buffer in the state as
+        // an array of big-endian u64s, converting them
+        // to machine endianness.
```

Comment:
Isn't it the other way round: We always make sure that this buffer is little-endian and now we are making sure to convert it to machine endianess?